landing/src/landing/Landing.tsx

```diff
@@ -1,35 +1,62 @@
+import reops2560WebP from '../assets/reops-2560.webp'
+import reops800JPEG from '../assets/reops-800.jpg'
+import reops1280JPEG from '../assets/reops-1280.jpg'
+import reops2560JPEG from '../assets/reops-2560.jpg'
 
 export default function Landing(): ReactElement {
-
     const navigate = useNavigate()
 
     return (
-        <div className='bg-[#75c4c5] h-[calc(100vh-148px)]'>
-            <div className='flex flex-col items-center py-12 space-y-4'>
-                <Heading size='xlarge'>Gi ditt samtykke</Heading>
-                <div className='text-center px-64'>
-                    <Heading size="small">
-                        I samtykke-løsningen kan du gi ditt digitale samtykke til
-                        brukertester og innsiktsarbeid for forbedringen av NAVs tjenester
+        <div className="bg-[#75c4c5] px-4 py-12 flex-1">
+            <div className="max-w-screen-xl mx-auto">
+                <div className="max-w-prose mx-auto text-center grid gap-4 justify-items-center">
+                    <Heading level="1" size="xlarge">
+                        Gi ditt samtykke
                     </Heading>
+                    <Ingress className="text-center">
+                        I samtykke-løsningen kan du gi ditt digitale samtykke
+                        til brukertester og innsiktsarbeid for forbedringen av
+                        NAVs tjenester
+                    </Ingress>
+                    <Button
+                        icon={<Right aria-hidden />}
+                        iconPosition="right"
+                        onClick={() => navigate('/innlogging')}
+                    >
+                        Til samtykke
+                    </Button>
                 </div>
-                <Button 
-                    icon={<Right aria-hidden />} 
-                    iconPosition="right"
-                    onClick={() => navigate('/innlogging')}
-                >
-                    Til samtykke
-                </Button>
+                <picture>
+                    <source
+                        srcSet={`${reops800AVIF} 800w, ${reops1280AVIF} 1280w`}
+                        sizes="(max-width: 1280px) 1280px, 100vw"
+                        type="image/avif"
+                    />
+                    <source
+                        srcSet={`${reops800WebP} 800w, ${reops1280WebP} 1280w`}
+                        sizes="(max-width: 1280px) 1280px, 100vw"
+                        type="image/webp"
+                    />
+                    <source
+                        srcSet={`${reops800JPEG} 800w, ${reops1280JPEG} 1280w`}
+                        sizes="(max-width: 1280px) 1280px, 100vw"
+                        type="image/jpeg"
+                    />
+                    <img
+                        src={reops800JPEG}
+                        alt="Illustrasjon av mennesker som bærer forskjellige geometrisk objekter."
+                    />
+                </picture>
```

Comment:
Koden for responsive bilder med flere formater og størrelser går helt sikkert an å gjøre smartere. Her satte jeg opp alt manuelt.